homeassistant/components/roku/media_player.py

```diff
@@ -234,6 +272,58 @@ async def search(self, keyword):
         """Emulate opening the search screen and entering the search keyword."""
         await self.coordinator.roku.search(keyword)
 
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """Implement the websocket media browsing helper."""
+        if media_content_type in [None, "library"]:
+            is_tv = self.coordinator.data.info.device_type == "tv"
+            return browse_media_library(channels=is_tv)
+
+        response = None
+
+        if media_content_type == "apps":
+            response = {
+                "title": "Apps",
+                "media_content_id": "apps",
+                "media_content_type": "apps",
+                "can_expand": True,
```

Comment:
I'm unsure here if you should be setting can expand, when you are already providing the children? Maybe somebody can shed some light on that.

Reply:
https://github.com/home-assistant/core/pull/39652#discussion_r483703549